main.py

```diff
@@ -0,0 +1,138 @@
+from pickle import FALSE
+import image_processor
+import camera
+import MainMovement
+import cv2
+import time
+import ref_commands
+from Color import Color
+from xbox360controller import Xbox360Controller
+
+
+motion = MainMovement.RobotMotion()
+
+def main_loop():
+
+    basketcolor = Color.BLUE # if no referee command is received we use this
+
+    debug = False   #Whether or not to show the debug frame
+
+    referee = ref_commands.Referee_cmd_client()
+    cam = camera.RealsenseCamera(exposure = 100)
+
+    try:
+        controller = Xbox360Controller(0, axis_threshold=0.2)
+        controller_connected = True
+    except:
+        controller_connected = False
+
+    #robotid for referee
+    robotID = "jurto"
+
+    #predefining first state
+    current_state = MainMovement.States.spin
+
+    #class call from other files
+    movementState = MainMovement.StateMachine(motion,cam.rgb_width,cam.rgb_height)
+    processor = image_processor.ImageProcessor(cam, debug=debug)
+    
+    #opening all that needs to be opened
+    referee.open()
+    motion.open()
+    #print(motion.ser)
+    processor.start()
+
+    start = time.time()
+    fps = 0
+    frame = 0
+    frame_cnt = 0
+
+    cv2.namedWindow('debug')
+    
+    
+    try:
+        #movementState.connect(ip,port)
+        status = False
+        #
+        while True:
+
+            #referee command reading
+            msg = referee.get_cmd()
+            
+            if msg is not None and msg['signal'] == 'start':
+                if msg["targets"][0] == robotID:
```

Comment:
If we are the only receivers of the command, then we are index 0, so it will always go into that block. It will only check further if we are not the only receivers and not on index 0.